src/pylast/__init__.py

```diff
@@ -2315,13 +2315,13 @@ def _get_friends():
 
         return _get_friends() if stream else list(_get_friends())
 
-    def get_loved_tracks(self, limit=50, cacheable=True, stream=True):
+    def get_loved_tracks(self, limit=50, cacheable=True, stream=False):
         """
         Returns this user's loved track as a sequence of LovedTrack objects in
         reverse order of their timestamp, all the way back to the first track.
 
         If limit==None, it will try to pull all the available data.
-        If stream=True, it will yield tracks as soon as a page has been retrieved.
+        If stream=False, it will yield tracks as soon as a page has been retrieved.
```

Comment:
Should this still say `If stream=True, ...`?

We can also remove the `stream=False` from test calls, it has no effect.